db/migrate/db/migrate/20220606061813_add_smtp_transaction_response_to_email_logs.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddSmtpTransactionResponseToEmailLogs < ActiveRecord::Migration[7.0]
+  def change
+    add_column :email_logs, :smtp_transaction_response, :string, null: true, limit: 500, index: true
```

Comment:
What is the cardinality of the response here going to be like? I'm thinking about whether an index is required here?

Reply:
It will be 1:1, we do not want an index.

Reply:
@Supermathie in the UI mockup there is a search field:

![image](https://user-images.githubusercontent.com/920448/173007321-cbeea9d0-391d-4fd1-aa19-c1fa51576771.png)

Which is why I added an index. If we don't need the index that's fine, just thought we would need it for `LIKE '%:smtp_id%'`

Reply:
> just thought we would need it for LIKE '%:smtp_id%'

This will likely result in a table scan so an index is not going to be helpful.

Reply:
You're right, have removed it now